_includes/did-you-know-box.html

```diff
@@ -0,0 +1,7 @@
+<div class="did-you-know-box">
+    <span class="did-you-know-description">
+        "Czy wiesz, że?" to seria krótkich artykułów Lorem ipsum dolor sit amet, consectetur adipisicing elit. Adipisci
```

Comment:
lorem? nice try...

Reply:
ja nie wiedziałem, że lorem ipsum :P

Reply:
Ze spokojem, to jeszcze nie jest gotowy PR :) Jak dostanę informację jaka treść tu ma być, to lorem zniknie